src/ucp/wireup/wireup.c

```diff
@@ -1101,19 +1110,18 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
                                    const ucp_unpacked_address_t *remote_address,
                                    unsigned *addr_indices)
 {
-    ucp_worker_h worker                            = ep->worker;
-    uint64_t tl_bitmap                             = local_tl_bitmap &
-                                                     worker->context->tl_bitmap;
-    ucp_rsc_index_t cm_idx                         = UCP_NULL_RESOURCE;
-    ucp_lane_index_t reuse_lane_map[UCP_MAX_LANES] = { UCP_NULL_LANE };
+    ucp_worker_h worker                  = ep->worker;
+    uint64_t tl_bitmap                   = local_tl_bitmap &
+                                           worker->context->tl_bitmap;
+    ucp_rsc_index_t cm_idx               = UCP_NULL_RESOURCE;
+    ucp_lane_index_t connect_lane_bitmap = UINT8_MAX;
```

Comment:
this should be initialized in ucp_wireup_check_config_intersect to ucs_mask(num_lanes)

Reply:
done